### DISCUSSION TITLE
No styles applied

### DISCUSSION THREAD

**Discussion by iamsubingyawali** [gilbitron/laravel-vue-pagination#137] [Help & support] [answered]
@gilbitron  Hi, I am simply implementing this package by importing within my component and using it there. Links work fine and pagination works fine as well but default styles are not applied. What might be the problem? I am getting output like this:

![image](https://user-images.githubusercontent.com/45819206/179442899-51b356ca-f17c-4f3b-b056-313b5bef839e.png)

**Comment by gilbitron** [comment, marked as answer]
Are you using Bootstrap?

**Reply by iamsubingyawali** [reply]
My mistake. I was missing Bootstrap dependency.

But is there any way to remove this 'current' text and show only the page number?

![image](https://user-images.githubusercontent.com/45819206/179827159-d45913a7-be41-43e9-b9a4-58aaf00af81f.png)
